docs/client_reference.rst

```diff
@@ -1561,6 +1568,21 @@ All exceptions are available as members of *aiohttp* module.
 
       Invalid URL, :class:`yarl.URL` instance.
 
+.. class:: ContentDisposition
+
+    Represent Content-Disposition header
+
+    .. attribute:: value
+
+    Value of Content-Disposition header itself, e.g. ``attachment``.
```

Comment:
``"attachment"`` - you want to notice that the value is a string typed.

Reply:
Will do